salt/modules/nilrt_ip.py

```diff
@@ -0,0 +1,571 @@
+# -*- coding: utf-8 -*-
+'''
+The networking module for NI Linux Real-Time distro
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+import time
+
+
+log = logging.getLogger(__name__)
+
+# Import third party libs
+try:
+    import pyconnman
+    import dbus
+except ImportError:
```

Comment:
What we probably want to do here is to move this into `__virtual__` and have that function return `False` if the `ImportError` is caught. That will ensure that Salt's loader system correctly displays the right error to the user and won't load the module when requested. There are many examples of this in various Salt execution modules.